packages/calling-stateful-client/src/Converter.ts

```diff
@@ -141,6 +141,10 @@ export function convertSdkCallToDeclarativeCall(call: CallCommon): CallState {
     /* @conditional-compile-remove(call-transfer) */
     transfer: {
       acceptedTransfers: {}
+    },
+    /* @conditional-compile-remove(optimal-video-count) */
+    optimalVideoCountFeature: {
+      optimalVideoCount: call.feature(Features.OptimalVideoCount).optimalVideoCount || 4
```

Comment:
Can we remove the `|| 4` here - if the SDK result can be undefined then we should have this as undefined and others layers can place defulat values when it doesn't exist. The state here should be a reflection of what is in the calling SDK